### PR TITLE
Desktop version: Milestone 1

### PR DESCRIPTION
**Pull request by saluumaa**
In this part I:
1. Added media query in the stylesheet to make my portfolio responsive
2. Added div holder to the Html file in the contact section
3. Added Navbar texts in the header section
4. Fix errors using linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,16 +6,19 @@
...
-        <img src="images/Union.png" alt="navigation icon">
+    <nav class="nav-bar">
+        <img src="images/Union.png" alt="navigation icon" class="img-nav">
+        <ul class="nav-list">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
     </nav>
     </header>
```
Comment (emmiiorji) [inline index.html:23]: Your design of the navigation bar looks great. I think that it'll also be great to make the navigation bar fixed as shown on the design template.

Your design:
image

Figma design:
image

File: style.css [hunk 2]
```diff
@@ -376,3 +390,177 @@ header a {
...
+  }
+
+  .section-one {
+    background-image: url('images/header-image.png');
+    background-position: top left;
+    background-size: 99% 100%;
+    background-color: #fff;
+    padding: 80px 140px;
+    font-family: 'Poppins', sans-serif;
+  }
```
Comment (emmiiorji) [inline style.css]: Kindly ensure that the hero section is as close to the design template as possible. From the design template, the height is at least 916px long. Also, there's more left and right margins between the text and  on the figma design than on your design. Kindly adjust to match. 

Your design:
image

Figma design:
image

File: style.css [hunk 3]
```diff
@@ -376,3 +390,177 @@ header a {
...
+
+  .works-section .card-reverse {
+    flex-direction: row-reverse;
+    border: 2px solid #a7aeff;
+  }
+
+  .card-img {
+    flex-basis: 30%;
+  }
```
Comment (emmiiorji) [inline style.css]: Kindly ensure that the cards are designed as closely to the figma design as possible. The image and description both take up equal width.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @saluumaa, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -14,7 +16,10 @@ header {
   align-items: center;
   padding: 15px 24px;
   background-color: #fff;
-  font-family: 'Poppins', sans-serif;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 72px;
```
Comment (Felix45) [inline style.css:22]: - [ Required ] Please consider adding a `z-index: 1` CSS rule on the styling of your `header` section to make it always be on top of other elements when a user is scrolling the page. I experienced the screenshot below when browsing the page at the `About` section of the page. You can fix this by adding this CSS rule to your header styling.

```CSS
  z-index: 1;
```

## Your design
Screenshot from 2023-01-19 12-36-16

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @saluumaa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 
- [ ] Descriptive pull request title and summary :heavy_check_mark: 
- [ ] Good descriptive commit messages :heavy_check_mark: 
- [ ] GitHub flow is correctly used :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @saluumaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image
## Highlights
- [ ] Good PR title ✔️ 
- [ ] Descriptive commit message ✔️ 
- [ ] Professional README ✔️ 
- [ ] All linter checks passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**